.gitlab-ci.yml

```diff
@@ -96,12 +96,10 @@ early_reindex:
   when: manual
   script:
     - make reindex
-  timeout: 13h 10m  # also see max_timeout in reindex.py
```

Comment:
I think there is a default timeout which is much shorter. If that is the case, this wouldn't work. Please investigate. If I am right, please reinstate this setting and make it a generous 24h. Otherwise, let me know.

Reply:
Correct, default timeout is 60 minutes.